### PR TITLE
To-do-list: Add and Remove task from To-do-List

### PR DESCRIPTION
**Pull request by abiola-arobieke**
In this milestone:

The UI design was parsed.
Index.js, Index.html, and index.css files were updated.
Methods for performing a basic CRUD were added to the project.
Data were fetched dynamically and saved in the local storage.
Data were updated and saved dynamically in the local storage on adding, editing, or deleting a task.

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @abiola-arobieke, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Since it is not a requirement for this project and you have not deployed the project, it will be nice to delete the `live demo section`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,47 +1,154 @@
...
+});
+
+// Delete completed task
+const delCheckedTask = document.getElementById('clear-task');
+delCheckedTask.addEventListener('click', () => {
+  if (task.length !== 0) {
+    for (let i = checkedItem.length - 1; i >= 0; i -= 1) {
+      task.splice(checkedItem[i], 1);
+    }
+    localStorage.setItem('allTask', JSON.stringify(task));
```
[the remark this answers is not in the document]
Comment (abiola-arobieke) [reply src/index.js:149]: Please I don't understand. The data were updated and saved in the local storage after editing. You will need to press enter once you are done with the editing.

[the remark this answers is not in the document]
Comment (prolajumokeoni) [reply src/index.js]: This is currently fixe when I edit a task it if updated in the local storage 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @abiola-arobieke 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 🥇  
  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**